deploy/helm/templates/conversion-patch-hook.yaml

```diff
@@ -0,0 +1,96 @@
+{{- if .Values.webhooks.conversionEnabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-crd-patch
+  labels:
+    {{- include "kubeblocks.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded"
+spec:
+  ttlSecondsAfterFinished: 0
+  backoffLimit: 2
+  template:
+    metadata:
+      name: {{ .Release.Name }}-crd-patch
+      labels:
+        {{- include "kubeblocks.labels" . | nindent 8 }}
+    spec:
+      {{- with .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "kubeblocks.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      restartPolicy: Never
+      containers:
+        - name: post-install-job
+          image: "{{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.tools.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command:
+            - sh
+            - -c
+            - |
+              set -e
+              # 1. check the already installed KubeBlocks
+              deployments=$(kubectl get deployments -A -l app.kubernetes.io/name=kubeblocks --no-headers)
+              OLD_IFS=$IFS
+              IFS=$'\n'
+              for line in $deployments; do
+                namespace=$(echo "$line" | awk '{print $1}')
+                name=$(echo "$line" | awk '{print $2}')
+                old_kb_json=$(kubectl get deployment ${name} -n ${namespace} -o json)
+                version=$(echo ${old_kb_json} | jq '.metadata.labels["app.kubernetes.io/version"]' | tr -d '"')
+                replicas=$(echo ${old_kb_json} | jq '.spec.replicas')
+                if [ $replicas -ne 0 ]; then
+                    echo "ERROR: You must stop the existing ${namespace}/${name} version (${version}) before installing version 1.0."
+                    exit 1
+                fi
```

Comment:
If kb not stopped, can we stop it in this scripts?

Reply:
need to stop kb before install 1.0 cr